spec/symbol_count_graph_spec.rb

```diff
@@ -34,8 +34,10 @@
       expect(graph.options[:columns]).to eq(10)
     end
     it 'accepts numbers as strings' do
-      graph = SymbolCountGraph.new({ x: "2" })
-      expect(graph.options[:columns]).to eq(10)
+      expect{ SymbolCountGraph.new({ x: "2" }) }.to_not raise_error(NoMethodError)
+    end
+    it 'raises an error when value is not an Integer' do
+      expect{ SymbolCountGraph.new({ x: '@$' }) }.to raise_error(ArgumentError)
```

Comment:
I added a spec to test this! `commit 79803b0e9ed9d82a53671e62babe0ff12732edf5`
one question about the terminology:
is a spec a single test or a test file?